src/components/service/dateFormatting.ts

```diff
@@ -0,0 +1,8 @@
+export default function formattingDate(date) {
+  const formatting = new Date(date).toLocaleString("en-GB", {
```

Comment:
aslo, the 'formatting' variable isn't necessary you can just return the expression